paparazzi-annotations/src/main/java/app/cash/paparazzi/annotations/PaparazziPreviewData.kt

```diff
@@ -0,0 +1,24 @@
+package app.cash.paparazzi.annotations
+
+import androidx.compose.runtime.Composable
+
+public sealed interface PaparazziPreviewData {
```

Comment:
I have a guess what this interface is for but I'm not positive. A comment would help explain its purpose.

Reply:
I'd go a bit further and say whatever is using this should also be part of this PR; otherwise, it's challenging to understand the intent behind this.

would you mind splitting up the PRs in a way that makes incremental development and review easier?

Reply:
yep, this should be in the a pr down the stack. Cleaning it up